src/main/webapp/styles/add-recommendation.css

```diff
@@ -57,7 +57,7 @@
   transition: all 300ms ease-in-out 0s;
 }
 
-/* Add Rec Form Elements */
+/* --- Add Rec Form Elements --- */
```

Comment:
There's no need to use the `---` before or after the comment, please use the [standard format](https://www.w3schools.com/css/css_comments.asp) instead :)

Reply:
Ah okay, this is a convention I used in the past, but I'll swap to standard format - thanks!